### PR TITLE
Update descriptions for filters 104, 243 and related ones

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/FiltersRegistry/issues/601

https://github.com/AdguardTeam/FiltersRegistry/issues/604

https://github.com/AdguardTeam/FiltersRegistry/issues/607

### CODE REVIEW

**Comment by aalpatkina** [comment]
Please also update this string:

"filter.104.description"
Chinese supplement for EasyList. --> Chinese supplement for EasyList. Already included in AdGuard Chinese filter.

**Review by aalpatkina — APPROVED** [review]

**Comment by zloyden** [comment]
Do not merge it. Waiting for translations.

**Review by Alex-302 — APPROVED** [review]

File: filters/ThirdParty/filter_107_EasyListGermany/metadata.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "filterId": 107,
   "name": "EasyList Germany",
-  "description": "German supplement for EasyList. Already included in AdGuard German filter.",
+  "description": "Additional filter list for websites in Germany. Already included in AdGuard German filter.",
```
Comment (aalpatkina) [inline filters/ThirdParty/filter_107_EasyListGermany/metadata.json:4]: Additional filter list for websites in German. Already included in AdGuard German filter.

File: locales/en/filters.json [hunk 2]
```diff
@@ -77,55 +77,55 @@
...
 	},
 	{
 		"filter.106.name": "EasyList Dutch",
-		"filter.106.description": "Dutch supplement for EasyList. Already included in AdGuard Dutch filter."
+		"filter.106.description": "Additional filter list for websites in Dutch. Already included in AdGuard Dutch filter."
 	},
 	{
 		"filter.107.name": "EasyList Germany",
-		"filter.107.description": "German supplement for EasyList. Already included in AdGuard German filter."
+		"filter.107.description": "Additional filter list for websites in Germany. Already included in AdGuard German filter."
```
Comment (aalpatkina) [inline locales/en/filters.json:100]: Germany --> German

**Review by aalpatkina — APPROVED** [review]

**Review by aalpatkina — APPROVED** [review]

**Review by slavaleleka — APPROVED** [review]

**Review by aalpatkina — APPROVED** [review]

**Pull request merged**